GOVERNANCE.md

```diff
@@ -0,0 +1,177 @@
+
+#### Examples of recognized contributions
+
+- **GitHub:** Submitting a merged pull request
+- **GitHub:** Filing a detailed bug report or RFC
+- **GitHub:** Updating documentation!
+- Helping people on GitHub, Discord, etc.
+- Answering questions on Stack Overflow, Twitter, etc.
+- Blogging, Vlogging, Podcasting, and Livestreaming about GUC projects
+- This list is incomplete! Similar contributions are also recognized.
+
+#### Privileges
+
+At this time we have no specifics for this role. As the community grows we anticipate privileges to come. If you have ideas here please let us know!
+
+#### Responsibilities
+
+This role does not require any extra responsibilities or time commitment. We hope you stick around and keep participating!
+
+If you're interested in reaching the next level and becoming a Maintainer, you can begin to explore some of those responsibilities in the next section.
+
+#### Nomination Process
+
+n/a
+
+### Level 2 - Maintainer
+
+The **Maintainer** role is available to any contributor who wants to join the team and take part in the long-term maintenance of plugins in the GUC.
+
+The Maintainer role is critical to the long-term health of GUC plugins. Maintainers act as the first line of defense when it comes to new issues, pull requests and Discord activity. Maintainers are most likely the first people that a user will interact with on Discord or GitHub.
+
+**A Maintainer is not required to write code!** Some Maintainers spend most of their time inside of Discord, maintaining a healthy community there. Maintainers can also be thought of as **Moderators** on Discord and carry special privileges for moderation.
+
+There is no strict minimum number of contributions needed to reach this level, as long as you can show **sustained** involvement over some amount of time (at least a couple of weeks).
+
+#### Recognized Contributions
+
+- **GitHub:** Contributing a new or existing Gatsby plugin
+- **GitHub:** Submitting non-trivial pull requests and RFCs
+- **GitHub:** Reviewing non-trivial pull requests and RFCs
+- **Discord:** Supporting users in Discord
+- **Discord:** Active participation in calls and other events
+- **GitHub + Discord:** Triaging and confirming user issues
+- This list is incomplete! Similar contributions are also recognized.
+
+#### Privileges
+
+- All privileges of the [Contributor role](#level-1---contributor), plus...
+- `@maintainer` role on [Discord](https://discord.gg/gwnqfzetjv)
+- New name color on Discord: **blue**
+- Invitation to the private `#maintainers` channel on Discord
+- Invitation to the `maintainers` team on GitHub
+- Ability to moderate Discord
+- Ability to push branches to the repo (No more personal fork needed)
+- Ability to review GitHub PRs
+- Ability to merge GitHub PRs
+- Define project direction and planning
+
+#### Responsibilities
+
+- Participate in the project as a team player.
```

Comment:
```suggestion
- Participate in the project as a team player
```

Reply:
The intention here is just for consistency - either a trailing `.` or no trailing `.` on list items